pyro/contrib/glmm/glmm.py

```diff
@@ -1,7 +1,8 @@
 from __future__ import absolute_import, division, print_function
 
 import warnings
-from collections import OrderedDict  # noqa: F401  # for docstrings
+from collections import OrderedDict
+from contextlib2 import ExitStack
```

Comment:
this will fail on py3. you need something like:
```
try:
    from contextlib import ExitStack  # python 3
except ImportError:
    from contextlib2 import ExitStack  # python 2
```

Reply:
Travis seems to be fine with it